qiskit/_quantumprogram.py

```diff
@@ -1056,29 +1054,29 @@ def _run_internal(self, qobj_list, wait=5, timeout=60, silent=True,
 
     def _jobs_done_callback(self, jobs_results):
         """ This internal callback will be called once all Jobs submitted have
-            finished. NOT every time a job has finished."""
-        results = []
-        for result, qobj in jobs_results:
-            results.append(Result(result, qobj))
-
+            finished. NOT every time a job has finished.
+            
+        Args:
+            jobs_results (list): list of Result objects
```

Comment:
Could you please also update the docstring for `JobProcessor.__init__()` at line 279 (the one that describes callback → results) to clarify it is a `list of Result objects`?

Reply:
ok